tensorflow/core/common_runtime/mkl_cpu_allocator.h

```diff
@@ -266,7 +265,11 @@ class MklCPUAllocator : public Allocator {
 
  private:
   // Hooks provided by this allocator for memory allocation routines from MKL
-
+  bool always_use_system_allocator_ = [] {
```

Comment:
Can we move the implementation of this lambda function to a .cc file? (`TF_ENABLE_ONEDNN_OPTS` is also in a .cc file.)